packages/code-du-travail-nlp/api/sem_search.py

```diff
@@ -0,0 +1,146 @@
+import json
+import tensorflow as tf
+import tensorflow_hub as hub
+import numpy as np
+import tf_sentencepiece
+import unidecode
+
+from typing import List
+
+content_path = "./data/content.json"
+stops_path = "./data/stops.txt"
+
+cache_path = "~/Downloads"
+
+stringvec = List[str]
+
+
+def add_slash(c):
+    c["slug"] = "/{}".format(c["slug"])
+    return c
+
+
+class SemSearch():
+
+    def __init__(self, content_path: str, stops_path: str):
+        print("loading data...")
+        with open(content_path, "r") as f:
+            content = list(filter(
+                lambda row: "text" in row and not "code_du_travail" in row.get('slug'), json.load(f)))
```

Comment:
en Python 3.4+, c'est recommandé d'utiliser les `not` de maniere plus semantique, ici groupé avec le `in`
```suggestion
                lambda row: "text" in row and "code_du_travail" not in row.get('slug'), json.load(f)))
```